dangerfile.js

```diff
@@ -2,7 +2,7 @@ import { message, danger, warn } from "danger";
 const prettier = require("prettier");
 const fs = require("fs");
 var path = require("path");
-
+message("Thanks for PRing! Someone will be along to check it shortly!");
```

Comment:
no pls

Reply:
How about I only show that if it's a first-time contributor?